server/src/main/java/org/elasticsearch/index/fielddata/plain/SortedSetOrdinalsIndexFieldData.java

```diff
@@ -78,12 +72,8 @@ public SortedSetOrdinalsIndexFieldData build(
         }
     }
 
-    protected final String fieldName;
-    private final IndexFieldDataCache cache;
-    private final CircuitBreakerService breakerService;
-    private final Function<SortedSetDocValues, ScriptDocValues<?>> scriptFunction;
-    private final ValuesSourceType valuesSourceType;
     private static final Logger logger = LogManager.getLogger(SortedSetOrdinalsIndexFieldData.class);
+    private final Function<SortedSetDocValues, ScriptDocValues<?>> scriptFunction;
```

Comment:
Does the superclass also have this member?

Reply:
Oops, yes this should only live in the superclass now.